<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>主界面:内卷模式</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCalendarWidget" name="_calendar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>331</height>
     </rect>
    </property>
    <property name="gridVisible">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QCalendarWidget::SingleSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonConfirm">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>确认</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonInport">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>导入课表</string>
    </property>
   </widget>
   <widget class="QLabel" name="_picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>601</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string>地图导引(请先选择日期)</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonModify">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>修改课表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_buttonPersonalize">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>个性化设置</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="_timeEdit">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>630</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="_table">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>401</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>690</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>690</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>690</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
